spring-integration-amqp/src/main/java/org/springframework/integration/amqp/inbound/AmqpInboundChannelAdapter.java

```diff
@@ -78,15 +80,31 @@ public String getComponentType() {
 	@Override
 	protected void onInit() {
 		this.messageListenerContainer.setMessageListener((ChannelAwareMessageListener) (message, channel) -> {
```

Comment:
Not related to this change, but how about to extract the real class for this Lambda?
Something similar what we do in other places like Kafka or Kinesis. 

I feel like it would be much easier to consult logs in case of StackTraces.